### DISCUSSION TITLE
Kan vi skippe Lerna-versjonering av portalen?

### DISCUSSION THREAD

**Discussion by wkillerud** [fremtind/jokul#2420] [Spørsmål]
Ser GitHub Actions går igjennom en lang sløyfe ved endringer i portalen, f. eks. ved blogging ([f. eks. her](https://github.com/fremtind/jokul/runs/4022938833?check_suite_focus=true))

Kommer det av at Lerna må inkludere portalen for å få oppdatert pakker i dependencies? Eller vil vi kunne bruke f. eks. `--no-private` for å hoppe over disse potensielt unødvendige "releasene"?

https://github.com/lerna/lerna/tree/main/commands/version#--no-private

**Comment by piofinn** [comment]
Stemmer for å hoppe over portalen! Strengt tatt hadde den ikke trengt å være regnet som en pakke av lerna en gang, siden den uansett aldri publiseres noe sted..

**Reply by piofinn** [reply]
Kan jo egentlig bare gjøre dette i `lerna.json` i så fall. Med mindre det tuller til symlinkene for Jøkul-pakkene da.

```diff
-    "packages": ["packages/*", "portal"],
+    "packages": ["packages/*"],
```
